components/docker-auth/cmd/authn/authentication.go

```diff
@@ -105,15 +133,130 @@ func main() {
 	tokenValidity, err := validateToken(token, certificateInUse)
 	if err != nil {
 		glog.Errorf("Token is not valid - %s", err)
-		glogFlushAndExit(extension.ErrorExitCode)
+		extension.LogErrorAndExit(extension.ErrorExitCode)
 	}
 	glog.Info("signature verified")
 
 	if tokenValidity {
 		glog.Info("user successfully authenticated")
-		glogFlushAndExit(extension.SuccessExitCode)
+		extension.LogErrorAndExit(extension.SuccessExitCode)
```

Comment:
Name of the function `extension.LogErrorAndExit` should change. Here we are passing a success exit code and seems to be contradicting with the function name